webapp/src/ts/components/filters/freetext-filter/freetext-filter.component.ts

```diff
@@ -45,9 +45,7 @@ export class FreetextFilterComponent implements OnDestroy, OnInit, AbstractFilte
     }
 
     this.globalActions.setFilter({ search: this.inputText });
-    // always force the search, so the user is taken from the report detail page to the list page on mobile,
-    // when clicking on a case_id link
-    this.search.emit(true);
```

Comment:
I checked this interaction, clicking on a report with a `case_id` and it's broken right now. Removing this further breaks that interaction.

Reply:
Since it's two different problems and I'm going to work on them in separate PRs. Once the other one is merged, I'll update this PR.

Reply:
Awesome, thanks a lot!

Reply:
I've sent a fix for [that issue](https://github.com/medic/cht-core/pull/8403), which doesn't conflict with this other solution so far.